projects/etos_suite_runner/src/etos_suite_runner/lib/suite.py

```diff
@@ -318,6 +319,67 @@ def sub_suite_environments(self) -> Iterator[dict]:
             self._record_exception(exc)
             raise exc
 
+    def _sub_suite_environments_from_kubernetes(self) -> Iterator[dict]:
+        """All sub suite environments from Kubernetes.
+
+        Each sub suite environment is an environment for the sub suites to execute in.
+        """
+        environments = []
+        timeout = time.time() + self.etos.config.get("WAIT_FOR_ENVIRONMENT_TIMEOUT")
+        while time.time() < timeout:
+            time.sleep(5)
```

Comment:
Is this wait needed in the beginning? If so, please add a comment why.

Reply:
I don't know, nor do I know why it was added before. This is just a copy-paste from the eiffel loop which does the same.